Readme.md

```diff
@@ -100,30 +103,19 @@ const accounts = await polkadot.System.Account.entries({ count: 10 }).run()
 
 ## Development Networks
 
-Let's modify the usage above to target our configured devnet.
-
-```diff
-- import { polkadot } from "@capi/polkadot"
-+ import { polkadotDev } from "@capi/polkadot-dev"
-
-- const accounts = await polkadot.System.Account
-+ const accounts = await polkadotDev.System.Account
-    .entries({ count: 10 })
-    .run()
-```
-
-To run code that depends on a devnet, use the `serve` command, followed by a
-"--" and your devnet-using command. In this case, we'll run the script
-(`main.js`) with Node.JS.
+During development, we may want to swap out the underlying connection with that
+of a devnet. This can be achieved with the `serve` command, followed by target
+specificity (in our case `dev`), a "--" and your devnet-using command. In this
+case, we'll run the script (`main.js`) with Node.JS.
```

Comment:
```suggestion
During development, we may want to swap out the underlying connection with that
of a devnet. This can be achieved via targets — by specifying alternate targets
in your `nets.ts` file, you can switch to them by wrapping your command with
`capi serve --target someTarget --`. For example:
```